MoreResistanceOrderPages/Src/MoreResistanceOrderpages/Classes/UIStrategyCard_Multi.uc

```diff
@@ -57,15 +52,35 @@ simulated function OnInit()
 		PrevFactionButton.bAnimateOnInit = false;
 		PrevFactionButton.InitButton(, class'UIPhotoboothReview'.default.m_strPrevious, PrevFactionPage);
 		PrevFactionButton.SetPosition(0, 500);
-		PrevFactionButton.SetDisabled(true);
+		PrevFactionButton.SetDisabled(true);*/
+
+		NextFactionButton = Spawn(class'UIStrategyPolicy_LeftRightButton', self);
+		NextFactionButton.InitButton(NextFactionPage);
+		NextFactionButton.SetOrigin(class'UIUtilities'.const.ANCHOR_TOP_RIGHT);
+		NextFactionButton.SetAnchor(class'UIUtilities'.const.ANCHOR_TOP_RIGHT);
+		NextFactionButton.SetY(281);
+
+		PrevFactionButton = Spawn(class'UIStrategyPolicy_LeftRightButton', self);
+		PrevFactionButton.InitButton(PrevFactionPage);
+		PrevFactionButton.MC.SetNum("_xscale", -100);
+		PrevFactionButton.SetY(281);
+		PrevFactionButton.SetX(PrevFactionButton.Width + 2);
+		PrevFactionButton.SetVisible(false);
+
+		NextFactions = Spawn(class'UIList', self);
+		NextFactions.InitList('ListNextPageFactions', 1780, 530, 130, 185);
+
+		PrevFactions = Spawn(class'UIList', self);
+		PrevFactions.InitList('ListPrevPageFactions', 0, 530, 150, 185);
```

Comment:
The Lists need to initialized before the InitDecks to show up on screen init instead of after changing pages